.github/workflows/test-flake.yml

```diff
@@ -0,0 +1,36 @@
+name: "Test"
+on:
+  pull_request:
+jobs:
+  compile-nix-flake:
+    name: 'Nix flake'
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-11]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: 'Check out code'
+        uses: actions/checkout@v2.3.4
+        with:
+          # Check out pull request HEAD instead of merge commit.
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: 'Install Nix'
+        uses: cachix/install-nix-action@v15
+        with:
+          extra_nix_config: |
+            substituters = http://cache.nixos.org https://cache.iog.io
+            trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+
+      - name: 'Install Cachix'
+        uses: cachix/cachix-action@v10
+        with:
+          name: runtimeverification
+          extraPullNames: kore
+          signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+
+      - name: 'Build K Framework'
+        run: GC_DONT_GC=1 nix build .
+
+      # - name: 'Test K Framework'
+      #   run: GC_DONT_GC=1 nix build .#test
```

Comment:
Why is this commented? Is it just to reduce CI load while we test the changes?

Reply:
yeah those tests take over an hour on the public runner I think. We either only run them on release or cut down on the tests ran?

Reply:
Seems reasonable to me - we could maybe open an issue to define a "core" set of tests that don't take ages to run, and comment / delete this code.

Reply:
https://github.com/runtimeverification/k/issues/2726